node/clients/oms.ts

```diff
@@ -57,7 +57,7 @@ export class OMSCustom extends OMS {
       invoice,
       {
         headers: {
-          VtexIdClientAutCookie: this.context.adminUserAuthToken,
+          VtexIdClientAutCookie: this.context.adminUserAuthToken ?? '',
```

Comment:
Why are we passing this as an empty string?

Reply:
Simple validation, maybe it's redundant, but it's a good practice to have it. But it is considering always the `adminUserAuthToken` and only if the `adminUserAuthToken` is `null` or `undefined` it will get an empty string.

Reply:
@edyespinal if you prefer, we can remove it, although it doesn't affect the feature itself.

Reply:
Same as above 😅